ssl/quic/cc_dummy.c

```diff
@@ -8,106 +8,120 @@
  */
 
 #include "internal/quic_cc.h"
+#include "internal/quic_types.h"
 
 typedef struct ossl_cc_dummy_st {
-    char dummy;
+    size_t max_dgram_len;
 } OSSL_CC_DUMMY;
 
-static OSSL_CC_DATA *dummy_new(OSSL_PARAM *settings, OSSL_PARAM *options,
-                               OSSL_PARAM *changeables)
+static OSSL_CC_DATA *dummy_new(OSSL_TIME (*now_cb)(void *arg),
+                               void *now_cb_arg)
 {
-    return OPENSSL_zalloc(sizeof(OSSL_CC_DUMMY));
+    OSSL_CC_DUMMY *d = OPENSSL_zalloc(sizeof(*d));
+
+    if (d == NULL)
+        return NULL;
+
+    d->max_dgram_len = QUIC_MIN_INITIAL_DGRAM_LEN;
+    return (OSSL_CC_DATA *)d;
 }
 
 static void dummy_free(OSSL_CC_DATA *cc)
 {
     OPENSSL_free(cc);
 }
 
-static void dummy_reset(OSSL_CC_DATA *cc, int flags)
+static void dummy_reset(OSSL_CC_DATA *cc)
```

Comment:
OSSL_UNUSED on the argument?
Sometimes we do do this, other times we don't.  So not critical.

Reply:
Probably not going to start adding this for everything since I haven't in the QUIC code up until now.